src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -0,0 +1,208 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import {
+  Bar,
+  ButtonIcon,
+  GU,
+  Header,
+  IconClose,
+  Layout,
+  Tabs,
+  breakpoint,
+  springs,
+  textStyle,
+  useTheme,
+  useViewport,
+} from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { useEsc } from '../../hooks/useKeyboardArrows'
+import Network from './Network/Network'
+
+const SECTIONS = new Map([['network', 'Network']])
+const PATHS = Array.from(SECTIONS.keys())
+const VALUES = Array.from(SECTIONS.values())
+
+const NETWORK_INDEX = 0
+
+const AnimatedDiv = animated.div
+
+function GlobalPreferences({ compact, onClose, onNavigation, sectionIndex }) {
+  const theme = useTheme()
+
+  useEsc(onClose)
+
+  const tabItems = VALUES.filter((_, index) => index === NETWORK_INDEX)
```

Comment:
We don't really need this bit—this was only used in the client to filter some tabs during onboarding.